onadata/libs/utils/logger_tools.py

```diff
@@ -105,6 +105,17 @@ def dict2xform(jsform, form_id, root=None):
         root, form_id, dict2xml(jsform))
 
 
+def get_first_record(queryset):
+    """
+    Returns the first item in a queryset sorted by id.
+    """
+    records = sorted([record for record in queryset], key=lambda k: k.id)
```

Comment:
is the queryset not already sorted by id at this point?

Reply:
Nope, that is being added by the `first()` method on the queryset and we do not have a default order on the `Instance` model which this function is mainly meant for.

Reply:
cool, I'm good with this